var/spack/repos/builtin/packages/perl/package.py

```diff
@@ -53,6 +53,7 @@ class Perl(Package):  # Perl doesn't use Autotools, it should subclass Package
     version('5.25.11', '37a398682c36cd85992b34b5c1c25dc1')
 
     # Maintenance releases (even numbers, recommended)
+    version('5.28.0', sha256='7e929f64d4cb0e9d1159d4a59fc89394e27fa1f7004d0836ca0d514685406ea8')
     version('5.26.2', 'dc0fea097f3992a8cd53f8ac0810d523', preferred=True)
```

Comment:
why is this still the preferred version?

Reply:
When making changes like this to upstream, I have to take into account how much it will affect packages that I am not trying to build. Just about everything depends on Perl (```spack dependents -t perl | wc -l```) as a build dependency and I haven't tested 5.28.0 on enough packages to confidently say it wont break builds.